sage-android/Sage/app/src/main/java/blueprint/com/sage/network/users/CreateUserRequest.java

```diff
@@ -0,0 +1,25 @@
+package blueprint.com.sage.network.users;
+
+import android.app.Activity;
+
+import com.android.volley.Response;
+
+import org.json.JSONObject;
+
+import blueprint.com.sage.models.User;
+import blueprint.com.sage.network.BaseRequest;
+
+/**
+ * Created by charlesx on 10/13/15.
+ */
+public class CreateUserRequest extends BaseRequest {
+    public CreateUserRequest(Activity activity, User user,
+                             Response.Listener<JSONObject> onSuccess, Response.ErrorListener onFailure) {
+        super(Method.POST, makeUrl("/users"), convertToParams(user, activity)
+        , (o) -> {
```

Comment:
Just a suggestion, but it might help to put all the JSON parsing here too so whenever a user calls this or any other endpoint they can just directly get the Java object and immediately do whatever they want with it